commands/lib/install.go

```diff
@@ -75,3 +75,29 @@ func installLibrary(lm *librariesmanager.LibrariesManager, libRelease *libraries
 	taskCB(&rpc.TaskProgress{Message: "Installed " + libRelease.String(), Completed: true})
 	return nil
 }
+
+//ZipLibraryInstall FIXMEDOC
+func ZipLibraryInstall(ctx context.Context, req *rpc.ZipLibraryInstallReq) (*rpc.ZipLibraryInstallResp, error) {
+	res := &rpc.ZipLibraryInstallResp{}
+	lm := commands.GetLibraryManager(req.GetInstance().GetId())
+	libPath := req.GetPath()
+	if err := lm.InstallZipLib(libPath); err != nil {
+		res.Status = "Error installing Zip Library"
+		return res, err
+	}
+	res.Status = "Success! Installed Zip Library"
+	return res, nil
+}
+
+//GitLibraryInstall FIXMEDOC
+func GitLibraryInstall(ctx context.Context, req *rpc.GitLibraryInstallReq) (*rpc.GitLibraryInstallResp, error) {
+	res := &rpc.GitLibraryInstallResp{}
+	lm := commands.GetLibraryManager(req.GetInstance().GetId())
+	url := req.GetUrl()
+	if err := lm.InstallGitLib(url); err != nil {
+		res.Status = "Error installing Git Library"
+		return res, err
```

Comment:
If there's an error don't return res, just the error. `return nil, err` is fine, same for `ZipLibraryInstall`.